### ISSUE TITLE
grunt breaks when using --gruntfile outside of the current working directory

### ISSUE THREAD

**Issue by ryanunger** [yatskevich/grunt-bower-task#19]
I am creating an SDK for a few projects, so that they can share grunt packages between them. Accordingly, I am calling grunt files from outside my base directory with my NPMs.

e.g. `grunt --gruntfile ../widgets/basic-text/build.js --base . bower:install`
### My grunt file is configured properly, per your docs

```
bower: {
    install: {
        options: { 
            targetDir: '../widgets/basic-text/src/lib',
            layout: 'byComponent',
            cleanup: true,
            verbose: true
        }
    }
}
```
### The Error

```
Running "bower:install" (bower) task
[notice] cleaning up Bower packages
[notice] cleaning up /Users/ryanunger/foobar/widgets/basic-text/src/lib

/Users/ryanunger/foobar/sdk/node_modules/grunt-bower-task/node_modules/bower/node_modules/tmp/lib/tmp.js:219
    throw err;
          ^
Error: Unable to read "/Users/ryanunger/foobar/sdk/component.json" file (Error code: ENOENT).
    at Object.util.error (/Users/ryanunger/foobar/sdk/node_modules/grunt/lib/grunt/util.js:57:39)
    at Object.file.read (/Users/ryanunger/foobar/sdk/node_modules/grunt/lib/grunt/file.js:237:22)
    at Object.file.readJSON (/Users/ryanunger/foobar/sdk/node_modules/grunt/lib/grunt/file.js:243:18)
    at BowerAssets.get (/Users/ryanunger/foobar/sdk/node_modules/grunt-bower-task/tasks/lib/bower_assets.js:17:32)
    at EventEmitter.module.exports (/Users/ryanunger/foobar/sdk/node_modules/grunt-bower-task/tasks/bower_task.js:76:40)
    at EventEmitter.emit (events.js:93:17)
    at module.exports (/Users/ryanunger/foobar/sdk/node_modules/grunt-bower-task/node_modules/bower/lib/commands/install.js:41:17)
    at EventEmitter.emit (events.js:96:17)
    at g (events.js:192:14)
    at EventEmitter.emit (events.js:93:17)
```
### My Guess...

I think the use of `cwd` in `./lib/bower_assets.js` is the root cause of this bug. **Can the location of the `component.json` file be an optional variable?**

**Comment by fantactuka** [comment]
I think it's possible to check for `grunt.option('base')` and use if instead of `cwd` if present

**Comment by fantactuka** [comment]
@ryanunger PR is merged, so if urgent you can go ahead and use `devel` branch to get things working

**Comment by yatskevich** [comment]
Fixed in version 0.2.1
